extensions/cms/package.json

```diff
@@ -224,7 +224,7 @@
           "description": "%cms.connectionOptions.encrypt.description%",
           "groupName": "%cms.connectionOptions.groupName.security%",
           "valueType": "boolean",
-          "defaultValue": null,
+          "defaultValue": "true",
```

Comment:
Are you planning on immediately following up with changing this to support boolean & numbers? Doing it in a separate PR makes sense but we should really fix this soon so we have context on all this still and it shouldn't be too hard to do so it'd be great to get that done while we're thinking about it.

Reply:
It's not a problem related to my changes and is an existing behavior. Moreover, there is no default value that is not a string as of now, so I don't think there is any noticeable impact. It's a nice to have but I don't think it meets the bar of urgency.

Reply:
At least make an issue and put it in an appropriate milestone to follow up on